tests/sparse_weights_test.py

```diff
@@ -76,7 +76,8 @@ def test_rezero_after_forward_1d(self):
             x = torch.ones((1,) + (in_features,))
             sparse(x)
 
-            # When training, the forward function should set weights back to zero.
+            # Rezero, verify the weights become sparse
+            sparse.rezero_weights()
```

Comment:
This test name is `test_rezero_after_forward` and it was designed to verify if `rezero_weights` was called after the `forward`. If we no longer call `rezero_weights` after `forward` then this test is obsolete